### PR TITLE
Vc 2301 start blank functional

### PR DESCRIPTION
**Pull request by orkunaybek**

### CODE REVIEW

**Review by pavelthq — APPROVED** [review]

**Review by pavelthq — APPROVED** [review]

File: public/components/startBlankFunctional/StartBlankPanel.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+const elementsStorage = vcCake.getStorage('elements')
+const cook = vcCake.getService('cook')
+const dataManager = vcCake.getService('dataManager')
+const settingsStorage = vcCake.getStorage('settings')
+let addedId, iframeWindow
+
+function startBlank (props) {
+  const startContainer = useRef()
+  useEffect(() => {
+    startContainer.current.classList.add('vcv-ui-state--visible')
```
Comment (konutis) [inline public/components/startBlankFunctional/StartBlankPanel.js:17]: Add class with react

File: public/components/startBlankFunctional/StartBlankPanel.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+  const localizations = dataManager.get('localizations')
+  const editorType = dataManager.get('editorType')
+  let type = editorType.replace('vcv_', '')
+  type = editorType === 'vcv_archives' ? 'Archive Page' : type.charAt(0).toUpperCase() + type.slice(1)
+  const headingPart1 = `${localizations ? localizations.blankPageTitleHeadingPart1 : 'Name Your '} ${type}`
+  const headingPart2 = localizations ? localizations.blankPageTitleHeadingPart2 : 'and Start Building'
+
+  const startBlankContent = (
+    <HfsPanelContent
```
Comment (konutis) [inline public/components/startBlankFunctional/StartBlankPanel.js:73]: move this to render

File: public/components/startBlankFunctional/lib/hsfPanelContent.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+  const handleSubmit = e => {
+    e && e.preventDefault()
+    onClick(inputValue.trim())
+  }
+
+  const handleTitleChange = e => {
+    e && e.preventDefault()
+    const value = e.currentTarget.value
+    setInputValue(value)
```
Comment (konutis) [inline public/components/startBlankFunctional/lib/hsfPanelContent.js:24]: you don't need this one

**Review by veidenbaums — APPROVED** [review]

**Comment by orkunaybek** [comment]
Closed due to unsolvable conflicts

**Pull request closed without merging**